fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/JDBCParameterBuildingVisitor.java

```diff
@@ -174,64 +174,42 @@ public boolean visit(String elementName, int elementIndex, Uri uri) {
     @Override
     public boolean visit(java.lang.String elementName, int elementIndex, Address address) {
         Parameter p;
-        
-        if (address.getCity() != null) {
+        for (com.ibm.fhir.model.type.String aLine : address.getLine()) {
```

Comment:
Is it safe to drop the  if (address.getLine() != null) check?

Reply:
yes.  our model doesn't allow null for repeating elements